paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/SwishDefinition.kt

```diff
@@ -25,11 +26,17 @@ internal object SwishDefinition : PaymentMethodDefinition {
     override fun uiDefinitionFactory(): UiDefinitionFactory = SwishUiDefinitionFactory
 }
 
-private object SwishUiDefinitionFactory : UiDefinitionFactory.RequiresSharedDataSpec {
-    override fun createSupportedPaymentMethod(sharedDataSpec: SharedDataSpec) = SupportedPaymentMethod(
+private object SwishUiDefinitionFactory : UiDefinitionFactory.Simple {
+    override fun createSupportedPaymentMethod() = SupportedPaymentMethod(
```

Comment:
does removing the sharedDataSpec from here imply that we now won't ever have a `lightThemeIconUrl` or `darkThemeIconUrl` for Swish? My understanding is that we could also get the sharedDataSpec from elements session, is there a way to update the swish `SupportedPaymentMethod` with the URL from that? Just trying to understand the implications of this

Reply:
Swish was never a LUXE LPM! Meaning we never got a spec from elements/sessions back. We relied only on the spec I removed from lpms.json.

Reply:
Yup! makes it clear it's not (and has never been) as supported LUXE LPM.